### ISSUE TITLE
Request: Support for build cuda support for libvmaf using ffnvcodec

### ISSUE THREAD

**Issue by ghost** [Netflix/vmaf#1220]
This will simplify native compilation and make cross-compilation possible.

The patch for ffmpeg vf_libvmaf_cuda seems to have been sent to ffmpeg-devel, and I don't think anyone wants to use both the cuda sdk and the ffnvcodec in a ffmpeg build.

**Comment by kylophone** [comment]
Yes, I think this is probably the best way forward, I was planning on getting to this eventually. If you (or anyone else) wants to work on it in the meantime, please let me know.

**Comment by gedoensmax** [comment]
So you need mvcc to build the CUDA code but it does not require CUDA runtime and linking against CUDA driver stubs that are for example included in a CUDA docker container works well.
